optimum/habana/accelerate/accelerator.py

```diff
@@ -220,6 +221,11 @@ def __init__(
             **kwargs,
         )
 
+        if self.state.is_fp8_enabled:
+            if self.fp8_recipe_handler is None:
+                self.fp8_recipe_handler = GaudiFP8RecipeKwargs()
+            self.fp8_recipe_handler = get_fp8_recipe(self.fp8_recipe_handler)
```

Comment:
In original accelerator.py code in this function is called from prepare_model() function. Please add a comment that the reason you moved this to __init__() is to cover both paths with and without deepspeed.

Reply:
Done